### PR TITLE
Add release 4.15.2 release note

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

Generates the 4.15.2 release note

### CODE REVIEW

File: site3/website/src/pages/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
 <!-- markdown-link-check-disable -->
 # Release notes
 
+## 4.15.2
+
+Release 4.15.2 contains various bug fixes and some dependencies CVE fixes.
+
+Apache BookKeeper users are encouraged to upgrade to 4.15.1.
```
Comment (StevenLuMT) [inline site3/website/src/pages/release-notes.md:8]: 4.15.1 version is error? @zymap

Comment (zymap) [inline site3/website/src/pages/release-notes.md:8]:
```suggestion
Apache BookKeeper users are encouraged to upgrade to 4.15.2.
```

File: site3/website/src/pages/release-notes.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+* AutoRecovery - Do not call shutdown() on the main ZookKeeper client thread. [PR #3487](https://github.com/apache/bookkeeper/pull/3487)
+* Check if channel closed before processing read request. [PR #3486](https://github.com/apache/bookkeeper/pull/3486)
+* Add missed call to onReadRequestFinish() when read request rejected. [PR #3482](https://github.com/apache/bookkeeper/pull/3482)
+* Fix the deadlock when only using io thread to handle request. [PR #3480](https://github.com/apache/bookkeeper/pull/3480)
+
+#### Metrics changes
+
+* Add latency stats for entry location index lookup so that possible RocksDB bottleneck can be detected. [PR #3444](https://github.com/apache/bookkeeper/pull/3444)
+
 ## 4.15.1
```
[the remark this answers is not in the document]
Comment (zymap) [reply site3/website/src/pages/release-notes.md:32]: What do you mean? This is the previous version

**Review by StevenLuMT — APPROVED** [review]
LGTM

**Review by nicoloboschi — APPROVED** [review]

**Review by hangc0276 — APPROVED** [review]

**Review by dlg99 — APPROVED** [review]

**Pull request merged**